edge/pkg/metamanager/process.go

```diff
@@ -128,6 +128,12 @@ func (m *metaManager) processInsert(message model.Message) {
 		return
 	}
 
+	msgSource := message.GetSource()
+	if msgSource == cloudmodules.DeviceControllerModuleName {
+		message.SetRoute(modules.MetaGroup, modules.DeviceTwinModuleName)
```

Comment:
why we need update the route info and not use the original?

Reply:
here the metamanager changes the source to `meta` so that the `classifyMsg` function of the devicetwin can recgonize the messages from meta and handle them as DMI message.